.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: Build
+
+on:
+    pull_request:
+    push:
+        branches: [main]
+        tags: ["*"]
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+    build_wheels:
+        name: Build wheel for ${{ matrix.os }}-${{ matrix.build }}${{ matrix.python }}-${{ matrix.arch }}
+        runs-on: ${{ matrix.os }}
+        strategy:
+            # Ensure that a wheel builder finishes even if another fails
+            fail-fast: false
+            matrix:
```

Comment:
This matrix is defining an outer product (with a couple of exclusions) of platforms we want to build on.